app/views/welcome/index.html.erb

```diff
@@ -8,7 +8,41 @@
 <div class="jumbotron text-center">
   <h1>Welcome to Knead Recipes!</h1>
 </div>
-<% unless current_user %>
+
+<% if current_user %>
+  <%= form_tag "/search" do %>
+    <h2>INGREDIENTS</h2>
+      <%= text_field_tag :ingredients, nil, placeholder: "Enter your ingredients:" %><br>
+    <h2>DIET</h2>
+      <%= check_box_tag 'diet[]', 'vegetarian' %>
+      <%= label_tag :vegetarian %><br>
```

Comment:
Just a heads up... I think we'll have pretty much all of these in a container of some sort and won't need the <br> tags.... I think...